_episodes/11-software-project.md

```diff
@@ -98,17 +99,20 @@ and then obtain a local copy of that project (from your GitHub) on your machine.
    ![View of your own fork of the software repository in GitHub](../fig/github-forked-repository-own.png){: .image-with-shadow width="900px" }
 
 > ## Exercise: Obtain the Software Project Locally
+>
 > Using the command line, clone the copied repository
 > from your GitHub account into the home directory on your computer using SSH.
 > Which command(s) would you use to get a detailed list of contents of the directory you have just cloned?
-> 
-> 
+>
+>
 > > ## Solution
 > > 1. Find the SSH URL of the software project repository to clone from your GitHub account.
 > > Make sure you do not clone the original repository but rather your own fork,
 > > as you should be able to push commits to it later on.
-> > Also make sure you select the **SSH** tab and not the **HTTPS** one -
-> > for this course, SSH is the preferred way of authenticating when sending your changes back to GitHub.
+> > Also make sure you select the **SSH** tab and not the **HTTPS** one.
+> > For this course, SSH is the preferred way of authenticating when sending your changes back to GitHub,
+> > but if you have already authenticated through HTTPS in the past,
+> > it is fine for you to continue using that method.
```

Comment:
We have removed the option to use HTTPS + auth tokens in the past and are asking students to use SSH for authentication. If we want to start supporting HTTPS again - we probably need instructions on how to cache the token in a local credential manager - which then complicates the setup (which also needs to be updated) and complicates running of the workshop (different authN methods, different issues, different credential managers).

Reply:
Following our in-person chat, I agree that for simplicity we should just stick with SSH since HTTPS is actually more complicated to manage now. I'll remove reference in an upcoming change.

Reply:
Addressed in [1efc1a0](https://github.com/carpentries-incubator/python-intermediate-development/pull/371/commits/1efc1a0f20dfe438722e1052363bd2d518496970)